src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,97 @@
+package pages;
+
+import common.AbstractComponent;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class HomePage extends AbstractComponent {
+
+    WebDriver driver;
+    WebDriverWait wait;
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy (css = "button[title='Close'][aria-label='Close']")
+    private List<WebElement> popUpButton1;
+
+    @FindBy(xpath = "//button[@class='popup-newsletter__close link']")
+    private List<WebElement> popUpButton2;
+
+    @FindBy (xpath = "//button[@class='exit-popup__close link']")
+    private List<WebElement> popUpButton3;
+
+//    @FindBy (css = ".product-item__title")
+//    private List<WebElement> livingRoomSets;
+//
+    @FindBy (xpath = "//div[@class='product-item product-item--vertical 1/4--lap 1/5--desk 1/6--wide is-selected']")
+    private List<WebElement> livingRoomSets;
+
+    private By roomSets = By.cssSelector("//div[@class='product-item product-item--vertical 1/4--lap 1/5--desk 1/6--wide is-selected']");
+    private By popUp1 = By.cssSelector("button[title='Close'][aria-label='Close']");
+    private By popUp2 = By.xpath("//button[@class='popup-newsletter__close link']");
+    private By popUp3 = By.xpath("//button[@class='exit-popup__close link']");
+
+    public void goTo(){
+        driver.get("https://decohubhome.com/");
```

Comment:
create static final variables for constants like "private static final MAIN_URL = https://decohubhome.com/";"

Reply:
I changed what you said but I'm not sure I'm doing it the right way